litmus-portal/frontend/src/pages/LoginPage/index.tsx

```diff
@@ -16,9 +16,20 @@ interface authData {
 const LoginPage = () => {
   const user = useActions(UserActions);
   const classes = useStyles();
+  let urlUser: string = '';
+
+  let urlPass: string = '';
+  if (window.location.hash.length > 0) {
```

Comment:
dont use window.location 
use the useLoaction() hook from react-router-dom that gives you these hash
like this
const {hash} = useLocation()